app/tracing.py

```diff
@@ -0,0 +1,19 @@
+import wrapt
+from aws_xray_sdk.core import xray_recorder
+from flask import current_app
+
+
+def trace():
+    @wrapt.decorator
```

Comment:
What is the advantage of this library over `functools.wraps` etc

Reply:
its what was used within the collector for xray_recorder, i can switch it to `functools.wraps` i dont really know which is better

Reply:
It looks like wrapt does the same stuff but better, I just wasn't sure whether something broke if you used the standard one. 

I don't mind leaving it in.